src/components/ContentListTable/components/StatusIcon.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { render } from '@testing-library/react';
+import StatusIcon from './StatusIcon';
+
+it('Render with Pending status', () => {
+  const { queryByText } = render(
+    <StatusIcon
+      status='Pending'
+    />,
+  );
```

Comment:
I was hoping to query for the actual icon here but I can't find a way to do that

Reply:
I think the text is enough here.